### ISSUE TITLE
Bug: Tiling Layout appearing when coming from gnome menu

### ISSUE THREAD

**Issue by LarsJaeger** [GnomeSnapExtensions/gSnap#34]
With Gnome 42 whenever I go from this menu (![image](https://user-images.githubusercontent.com/61934397/177118763-13eb736b-06d7-4452-a986-5f6b985e313e.png))
into the desktop mode (![image](https://user-images.githubusercontent.com/61934397/177118998-af7f6075-c487-4710-8004-080ce4d002d6.png)), the tiling view appears until i click on a window frame.
Installed through https://extensions.gnome.org/ and Firefox Add-on.

Edit: Somehow it differs, sometimes it only happens on the second workspace, sometimes on all the workspaces.

**Comment by meronz** [comment]
I cannot reproduce it by entering/exiting the overview mode, but I did reproduce a similar problem with enter/exit fullscreen when tabs are disabled. I will try to push a fix in a separate branch, so that you can test it and see if it works for you.

The PR you pushed (#36) only tries to circumvent the problem, introducing some other misbehaviour and I will not include it.

**Comment by meronz** [comment]
Oops, but I already fixed that in the main branch. Did you try to compile just the main branch (without your edits) to see if it works?
I will try and release this on EGO as soon as I can.

**Comment by LarsJaeger** [comment]
Sorry, my mistake... Yeah, using building from the master branch fixed it for me, I tried it before, but I think I forgot to reboot and only disabled and re-enabled the extension instead.
Actually I have absolutely no idea of Gnome and / or Typescript, so it seemed like it worked, although I had no idea what exactly I did there. 
However, the master branch is not the version you get when installing from the extension page, so you might need to update the version which is shipped there.
Thank you for your work, now that this is fixed it is my favorite extension.
